src/App.tsx

```diff
@@ -1,24 +1,154 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
+import React, { useCallback, useEffect, useState } from 'react';
+import { getTodos, addTodo, deleteTodo } from './api/todos';
+import { filteredTodos, activeTodosCount } from './helpers/functionsTodos';
+import { NotificationTodo } from './Components/NotificationTodo';
+import { TodoFooter } from './Components/TodoFooter';
+import { TodoHeader } from './Components/TodoHeader';
+import { TodoList } from './Components/TodoList';
+import { Loader } from './Components/Loader';
 
-const USER_ID = 0;
+import { Todo } from './types/Todo';
+import { Filter } from './types/Filter';
+import { Errors } from './types/Errors';
+
+const USER_ID = 6211;
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
-  }
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [isSelected, setIsSelected] = useState(true);
+  const [filter, setFilter] = useState(Filter.All);
+  const [error, setError] = useState(false);
+  const [errorNotification, setErrorNotification] = useState(Errors.Empty);
+  const [inputValue, setInputValue] = useState('');
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [deletingId, setDeletingId] = useState<number[]>([]);
+
+  const visibeTodos = filteredTodos(todos, filter);
+  const activeTodos = activeTodosCount(todos).length;
+  const completedTodos = visibeTodos.length - activeTodos;
+  const completedTodoList = filteredTodos(todos, Filter.Completed);
+
+  useEffect(() => {
+    const loadTodos = async () => {
+      try {
+        const loadedTodos = await getTodos(USER_ID);
+
+        setTodos(loadedTodos);
+      } catch (err) {
+        throw new Error();
+      } finally {
+        setIsSelected(false);
+      }
+    };
+
+    loadTodos();
+  }, []);
+
+  const setNotification = (message: Errors) => {
+    setErrorNotification(message);
+    setError(true);
+  };
+
+  const clearNotification = useCallback(() => {
+    setError(false);
+  }, []);
+
+  const clearInput = () => {
+    setInputValue('');
+  };
+
+  const handleCreateTodo = useCallback((title: string) => {
+    clearNotification();
+
+    if (!title) {
+      setNotification(Errors.Empty);
+    }
+
+    if (title.trim()) {
+      const newTodo = {
+        id: 0,
+        title,
+        userId: USER_ID,
+        completed: false,
+      };
+
+      setTempTodo(newTodo);
+      addTodo(USER_ID, newTodo)
+        .then((todo) => {
```

Comment:
please follow 1 code style
use `async/await` or `Promise` syntax
don't mix it in 1 file